pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/readerwriter/ValueReaderComparisonTest.java

```diff
@@ -156,22 +156,27 @@ void testSuperiorPrefixes(ValueReader reader, int numBytesPerValue, String... st
     void testConsistent(ValueReader reader, int numBytesPerValue, int numValuesToCompare) {
       for (int i = 0; i < numValuesToCompare; i++) {
         byte[] bytes = new byte[ThreadLocalRandom.current().nextInt(numBytesPerValue * 2)];
-        assertConsistentUtf8Comparison(reader, i, numBytesPerValue, new String(bytes, StandardCharsets.UTF_8));
-        for (int j = 0; j < 128; j++) {
+
+        for (int j = 1; j < 128; j++) {
```

Comment:
e.g. all zero fill data are not considered to be valid?

Reply:
Yeah, we skipped them in the test